.github/workflows/continuous-integration.yml

```diff
@@ -129,6 +129,9 @@ jobs:
         run: echo ::set-output name=app_folders::$(node script/github-actions/get-changed-apps.js --output-type folder)
         env:
           CHANGED_FILE_PATHS: ${{ steps.changed-files.outputs.all_changed_files }}
+ 
+      - name: Install dd-trace
+        run: yarn add dd-trace --dev
```

Comment:
`dd-trace` needs to be added to `package.json` instead. This will prevent unit tests from running locally because the mocha config won't be able to find the package

Reply:
tried that @theo and I am getting blocked by yarn.lock within the action

Reply:
so, I figured this has to do with something within my local setup with npm and yarn. I was able to include this now to the `package.json` and removed from the CI workflow.